oauthenticator/google.py

```diff
@@ -105,6 +105,21 @@ def _userdata_url_default(self):
         """,
     )
 
+    allow_nested_groups = Bool(
+        config=True,
+        help="""
+        Allow members of nested Google groups to sign in and/or administer
+        TLJH. If `True` the authenticator will recursively check a user's 
```

Comment:
Tljh mentioned but shouldnt be, as it can be used for any jupyterhub deployment, tljh/z2jh/no-dostribution etc

Reply:
Found it's way in via osmosis so to speak, as I used TLJH as my main testing. Will remove!